src/signing_key.rs

```diff
@@ -109,7 +109,9 @@ impl From<[u8; 32]> for SigningKey {
 impl zeroize::Zeroize for SigningKey {
     fn zeroize(&mut self) {
         self.seed.zeroize();
-        self.s.zeroize()
+        self.s.zeroize();
+        self.prefix.zeroize();
+        self.vk.zeroize();
```

Comment:
Should we use `derive(Zeroize)` to avoid doing this explicitly? https://docs.rs/zeroize/latest/zeroize/#custom-derive-support

Reply:
Done! f1a87ff5c66973754e3d6b69143ad200b2e7d025